addons/account/static/src/components/x2many_buttons/x2many_buttons.js

```diff
@@ -0,0 +1,55 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { standardFieldProps } from "@web/views/fields/standard_field_props";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+
+class X2ManyButtons extends Component {
+    static template = "account.X2ManyButtons";
+    static props = {
+        ...standardFieldProps,
+        treeLabel: String,
```

Comment:
I prefer the object spec
```suggestion
        treeLabel: { type:String },
```